site/index.html

```diff
@@ -34,6 +34,7 @@
         loadSidebar: "sidebar.md",
         alias: {
           "/installation/*.*/sidebar.md": "/sidebar.md",
+          "/book/*.*/sidebar.md": "/book/sidebar.md",

```

Comment:
I believe this is the source of the separate sidebar, I am not sure why we need several sidebars.  The overall content would not be too huge if we just had a collapsable TOC.

Reply:
Pending question: Should the sidebar of book pages be exclusively the outline of the book (as is the case with https://doc.rust-lang.org/stable/book/) with a link to the main page? Should it be as the rest of the main page currently is, with the only difference in browsing book pages being expanded entries in the sidebar for each chapter?

Reply:
I like to see a preview of this before deciding.

Reply:
Collapsible ToC merged in.